classes/class-email-service.php

```diff
@@ -174,33 +187,65 @@ public function action( $action, Backup $backup ) {
 					$error_message = ' - ' . $error_message;
 				}
 
-				$subject = sprintf( __( 'Backup of %s Failed', 'backupwordpress' ), $domain );
-
-				$message = sprintf( __( 'BackUpWordPress was unable to backup your site %1$s.', 'backupwordpress' ) . "\n\n" . __( 'Here are the errors that we\'ve encountered:', 'backupwordpress' ) . "\n\n" . '%2$s' . "\n\n" . __( 'If the errors above look like Martian, forward this email to %3$s and we\'ll take a look', 'backupwordpress' ) . "\n\n" . __( "Kind Regards,\nThe Apologetic BackUpWordPress Backup Emailing Robot", 'backupwordpress' ), home_url(), $error_message, 'backupwordpress@hmn.md' );
+				$subject = wp_strip_all_tags( sprintf(
+					/* translators: Domain URL */
+					__( 'Backup of %s failed', 'backupwordpress' ),
+					$domain
+				) );
+
+				$message = sprintf(
+						esc_html__(
+								'BackUpWordPress was unable to backup your site %1$s.' . "\n\n" .
+								'Here are the errors that we\'ve encountered: %2$s' . "\n\n" .
+								'If the errors above look like Martian, forward this email to %3$s and we\'ll take a look.' . "\n\n", 'backupwordpress' ),
+						esc_url( home_url() ),
+						$error_message,
+						'backupwordpress@hmn.md'
+				);
```

Comment:
@johnbillion How does this look? Addressing your feedback that the whole message should be a one string.